src/cider/nrepl/middleware/test.clj

```diff
@@ -67,10 +67,12 @@
 (defn- print-object
   "Print `object` using pprint or a custom print-method, if available."
   [object]
-  (let [print-fn (if (= (get-method print-method (:type (meta object)))
-                        (get-method print-method :default))
-                   pp/pprint
-                   println)]
+  (let [custom-type (:type (meta object))
+        print-fn (if (and custom-type
```

Comment:
I don't understand much what's going on here. 

* What's a `custom-type`?
  * i.e. worth adding a comment
* why can custom types result in a vanilla `println`?
  * I'm not sure of how 'customness' relates to the choice of `println`.

Reply:
See #683 for more details on that. I’m not sure myself, to be honest. I don’t understand how exactly the `:type` meta attribute gets assigned to a value, but I don’t want to repeat #683’s mistakes and break the formatting for that case. Basically, my goal here is to keep the existing tests passing.